crates/prql-compiler/tests/integration/main.rs

```diff
@@ -35,7 +35,13 @@ trait IntegrationTest {
     fn get_connection(&self) -> Option<DbConnection>;
     // We sometimes want to modify the SQL `INSERT` query (we don't modify the
     // SQL `SELECT` query)
-    fn import_csv(&mut self, protocol: &mut dyn DbProtocol, csv_name: &str, runtime: &Runtime);
+    fn import_csv(
+        &mut self,
+        protocol: &mut dyn DbProtocol,
+        csv_dir: &str,
+        csv_name: &str,
```

Comment:
These two could be a single path and then we extract the dir & name from it (but not a big deal)